plugin/trino-redshift/pom.xml

```diff
@@ -208,6 +208,7 @@
                                 <exclude>**/TestRedshiftTableStatisticsReader.java</exclude>
                                 <exclude>**/TestRedshiftTypeMapping.java</exclude>
                                 <exclude>**/Test*FailureRecoveryTest.java</exclude>
+                                <exclude>**/Test*FailureRecoverySmokeTest.java</exclude>
```

Comment:
`Test*FailureRecoveryTest.java` remains excluded; where is it icluded?

Reply:
thanks - i've documented now in the `fte-tests` profile a similar fashion as for the smoke tests `cloud-tests` profile.